exercises/simple-linked-list/example.py

```diff
@@ -0,0 +1,67 @@
+class Element(object):
+    def __init__(self, value):
+        self.value = value
+        self.next = None
+
+
+class LinkedList(object):
+    def __init__(self, head=None):
+        self.head = head
+        self.size = 0
+
+    def push(self, new_element):
+        new_e = Element(new_element)
+
+        if self.empty():
+            self.head = new_e
+        else:
+            new_e.next = self.head
+            self.head = new_e
+        self.size += 1
+
+    def pop(self):
+        if self.empty():
+            return None
+        else:
+            e = self.head
```

Comment:
No need for undescriptive short variable names, there is enough space for `element = self...`.